src/Transition/CreateNewIDFUsingRulesV8_3_0.f90

```diff
@@ -321,29 +339,27 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               SELECT CASE (MakeUPPERCase(TRIM(IDFRecords(Num)%Name)))
 
               CASE ('VERSION')
-                IF ((InArgs(1)(1:3) == '8.3').and. ArgFile) THEN
+                IF ((InArgs(1)(1:3)) == '8.3' .and. ArgFile) THEN
                   CALL ShowWarningError('File is already at latest version.  No new diff file made.',Auditf)
                   CLOSE(diflfn,STATUS='DELETE')
                   LatestVersion=.true.
                   EXIT
                 ENDIF
                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
-                OutArgs(1)='8.3'
+                OutArgs(1) = sVersionNum
                 nodiff=.false.
 
-!    !!!    Changes for this version
-   ! These will be conflicted, but uncommented when that branch is merged to develop
-   !           CASE('CHILLER:ELECTRIC:REFORMULATEDEIR')
-   !             nodiff=.false.
-   !             CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
-   !             ! fields 1-9 aren't affected
-   !             OutArgs(1:9)=InArgs(1:9)
-   !             ! add a blank for the new curve type field
-   !             OutArgs(10)=blank
-   !             ! then we just push the rest of the fields down 1
-   !             OutArgs(11:)=InArgs(10:)
-   !             CurArgs = CurArgs + 1
-                
+    !!!    Changes for this version
+              CASE('CHILLER:ELECTRIC:REFORMULATEDEIR')
+                nodiff=.false.
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                ! fields 1-9 aren't affected
+                OutArgs(1:9)=InArgs(1:9)
+                ! add a blank for the new curve type field
```

Comment:
@Myoldmopar Any reason not to have transition insert the key choice here?  Much clearer to put "LeavingCondenserWaterTemperature" here.